src/ScoreSystem.js

```diff
@@ -0,0 +1,27 @@
+
+export class ScoreSystem {
+	constructor() {
```

Comment:
Although why not just create a score keeping counter for a specific player character? If we move the player character creation to Logic, it can create and manage the scorekeeping stuff there.